src/pages/Home/home.scss

```diff
@@ -0,0 +1,124 @@
+/* stylelint-disable */
+@import "../../common.scss";
+
+.home-container {
+  background-color: $secondaryColor;
+  position: relative;
+  color: $font-primary;
+
+  .top {
+    display: flex;
+    gap: 24px;
+    align-items: center;
+    padding: 10px;
+    min-height: 20vh;
+    box-shadow: $box-shadow;
+    margin-bottom: 25px;
+
+    .left {
+      flex: 1;
+      justify-self: end;
+
+      img {
+        float: right;
+        width: $img-width;
+        height: $img-height;
+
+        @include mobile {
+          width: 100%;
+          height: 60%;
+        }
+      }
+    }
+
+    .right {
+      flex: 1;
+      align-self: start;
+      text-align: center;
+      padding: 20px;
+      display: flex;
+      flex-direction: column;
+      border-left: #999 2px solid;
+      gap: 12px;
+
+      p {
+        font-size: 20px;
+        font-weight: 400;
+      }
+    }
+  }
+
+  .bottom {
+    display: flex;
+    flex-direction: column;
+
+    h2 {
+      text-align: center;
+      text-transform: capitalize;
+    }
+
+    .search {
+      display: flex;
+      align-items: center;
+      margin: 24px auto;
+      width: 60vw;
+      padding: 10px;
+      border-radius: 50px;
+      background-color: $search-bg;
+
+      input {
+        padding: 5px;
+        outline: none;
+        border: none;
+        width: 100%;
+        font-size: 25px;
+        font-weight: 400;
+        background-color: transparent;
+      }
+    }
+
+    .items {
+      display: grid;
+      grid-template-columns: 1fr 1fr;
+      padding: 5px;
+
+      .item {
+        display: flex;
+        justify-content: space-between;
+
+        @include small {
+          font-size: 13px;
+          padding: 5px;
+        }
+
+        @include mobile {
+          font-size: 15px;
+          padding: 5px;
+        }
+
+        font-size: 20px;
+        border: 2px solid #999;
+        padding: 10px;
+        margin: 3px;
+
+        &:hover {
+          transform: scaleX(0.97);
+        }
+
+        .item-container {
+          @include mobile {
+            padding: 5px;
+          }
+          padding: 15px;
+          margin: 5px;
+          text-align: left;
+          p {
+            color: $font-primary;
+            font-weight: 400;
+            margin-bottom: 5px;
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
- [ ] Good job so far with your project it looks great and works as expected :clap:  But we have an issue with your design :cry: As you check on the project requirement, we expect you to work on mobile design and we can see some differences between your design and the original design. Please try to make your design similar to the original design for both the home and details page as a developer it is important to pay attention to details :+1: 

#### Figma design
![metrics_app](https://user-images.githubusercontent.com/6615394/195218382-33f7f77a-f88f-41ac-a8ce-795156a3a495.jpg)

#### You design

![Screenshot from 2022-10-14 23-34-51](https://user-images.githubusercontent.com/6615394/195957837-b0bf51c0-7399-44f8-b1e7-110d672582a6.png)

![Screenshot from 2022-10-14 23-36-47](https://user-images.githubusercontent.com/6615394/195957928-9ffb845b-403d-40d1-b1ff-f05f5eb6f8f3.png)

- [ ] Please make sure to follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), especially the layout: composition and space between elements as requested for this project :+1:

Reply:
Hello, @codecaiine  Thank you for your feedback! do you mean I have to use a background image?

Reply:
Hello @melashu  I have realized that you might not have fully understood the previously requested changes I will try to elaborate to you as best as I can.

- [ ]  The theme that you have chosen for the project is a dark-blue-ish color theme which does not match i any way the color schemes that are required for this project. You have two themes that you can choose from one being a pink-ish color theme and the other one is a sky-blue-ish color theme. The requested changes by the [previous reviewer](https://github.com/melashu/livecrypto/pull/1#discussion_r996214252) echoed this to you. Kindly change the color theme that you chose for your design and kindly use one of the two provided color theme alternatives. This way, your project will match the Behance design and your project will be ready for approval.

**This is one of the desired themes for this project:**

![image](https://user-images.githubusercontent.com/86318284/195975961-03facc4a-ef26-433c-aff0-0e25200931d7.png)

**This is the other theme that you can choose from:**

![image](https://user-images.githubusercontent.com/86318284/195976141-d3ba1f7d-68ef-4893-9075-2e8f1f6d0b0c.png)

**Now here is your design currently:**

![image](https://user-images.githubusercontent.com/86318284/195976158-c63534f8-e18b-46ef-9f87-99c3d5b89346.png)

**Note the contrast?** Kindly ensure that you make this change and your project will be ready for approval.